app/src/main/java/edu/uc/it3048/mypokedex/ProfileScreenActivity.kt

```diff
@@ -41,15 +41,16 @@ class ProfileScreenActivity : AppCompatActivity() {
     private val CAMERA_REQUEST_CODE = 918
     private val GALLERY_REQUEST_CODE = 555
     private lateinit var loginProviders : List<AuthUI.IdpConfig>
-    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
-    private var storageReference = FirebaseStorage.getInstance()
     private lateinit var currentPhotoPath : String
+    private var firestore : FirebaseFirestore = FirebaseFirestore.getInstance()
     private var selectedPhotoUri : Uri? = null
 
     init {
         firestore.firestoreSettings = FirebaseFirestoreSettings.Builder().build()
     }
 
+    //region On Functions
```

Comment:
I added 3 different regions in this file for better organization of functions that are related to similar things. Here there are 3 functions that start with "on" so I grouped them in a region. Regions are collapsable so they can be nice to have, but ultimately do not affect functionality.

Reply:
I don't recommend re-ordering someone else's code.
- They likely wrote it in a way that makes sense to them.
- Bigger issue: it makes merges difficult, as four people may be working on the same file, and then one person re-orders it, and VCS doesn't know how to merge it.